_posts/2024-01-02-release.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Release 2024.01
+author: Priya
+date: 2024-01-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+#### Project Locking for Efficient Data Flow Schedules
```

Comment:
#### Removed redundant runs when two or more data pulls are close in time
Mammoth now plans the running of pipelines for those cases where data pulls are very close in time to each other. Earlier there was a possibility that each one triggers its independent run leading to redundant procssing.